src/scrape_up/googleNews/googleNews.py

```diff
@@ -85,3 +86,26 @@ def getArticles(self):
             }
             ejson = json.dumps(error_message)
             return ejson
+    def Top_stories(self):
+     """
+    Class - `GoogleNews`
+    Example:
+    ```
+    # articles = GoogleNews("github")
+    ```
+    Prints the top stories from Google News RSS feed.
+    """
+    url = "https://news.google.com/news/rss"
+    try:
+        page = requests.get(url)
+        soup = BeautifulSoup(page.content, features="xml")
+        titles = soup.find_all("title")
+        if len(titles) > 0:
+            for title in titles:
+                print(title.text.upper())
+                print(time.ctime())
+                print("-" * 70)
```

Comment:
Why are you printing the values, return the data in JSON format.

Check the above function, how data is returned as JSON output and None in case it fails.

Reply:
ok